### PR TITLE
fix: Prop spreading for Time

### PR DESCRIPTION
**Pull request by cjskillingstad**
- Spread props to Time's elements
- Update unit tests
- Update docs

### CODE REVIEW

File: src/Time/Time.Component.js [hunk 1]
```diff
@@ -49,6 +49,54 @@ export const TimeComponent = () => {
...
                         description:
                             'object - The time component values , contains four properties: hour with values from 01 to 12 when format12Hours is true or 00 to 23 when format12Hours is false, minute with values from 00 to 59, second with values from 00 to 59, meridiem with values 0(am), 1(pm)'
+                    },
+                    {
+                        name: 'hoursDownButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s hours down button element.'
+                    },
+                    {
+                        name: 'hoursUpButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s hours down button element.'
```
Comment (greg-a-smith) [inline src/Time/Time.Component.js]: Should be "hours up button"

File: src/Time/Time.Component.js [hunk 2]
```diff
@@ -49,6 +49,54 @@ export const TimeComponent = () => {
...
+                        name: 'hoursInputProps',
+                        description: 'object - Additional props to be spread to the Time component\'s hours input element.'
+                    },
+                    {
+                        name: 'minutesDownButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s minutes down button element.'
+                    },
+                    {
+                        name: 'minutesUpButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s minutes down button element.'
```
Comment (greg-a-smith) [inline src/Time/Time.Component.js]: Should be "minutes up button"

File: src/Time/Time.Component.js [hunk 3]
```diff
@@ -49,6 +49,54 @@ export const TimeComponent = () => {
...
+                        name: 'minutesInputProps',
+                        description: 'object - Additional props to be spread to the Time component\'s minutes input element.'
+                    },
+                    {
+                        name: 'secondsDownButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s seconds down button element.'
+                    },
+                    {
+                        name: 'secondsUpButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s seconds down button element.'
```
Comment (greg-a-smith) [inline src/Time/Time.Component.js]: Should be "seconds up button"

File: src/Time/Time.Component.js [hunk 4]
```diff
@@ -49,6 +49,54 @@ export const TimeComponent = () => {
...
+                        name: 'secondsInputProps',
+                        description: 'object - Additional props to be spread to the Time component\'s seconds input element.'
+                    },
+                    {
+                        name: 'meridiemDownButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s meridiem down button element.'
+                    },
+                    {
+                        name: 'meridiemUpButtonProps',
+                        description: 'object - Additional props to be spread to the Time component\'s meridiem down button element.'
```
Comment (greg-a-smith) [inline src/Time/Time.Component.js]: Should be "meridiem up button"

Comment (cjskillingstad) [reply src/Time/Time.Component.js]: Oops, copy/paste mistake. Fixed in 7496561

**Pull request merged**